misc/tutorial/210_selection.ngdoc

```diff
@@ -207,4 +207,26 @@ auto-selects the first row once the data is loaded.
       height: 400px;
     }
   </file>
+  <file name="scenario.js">
+    var gridTestUtils = require('../../test/e2e/gridTestUtils.spec.js');
+    var GridObjectTest = require('../../test/e2e/gridObjectTestUtils.spec.js');
+    var grid1Selection = new GridObjectTest('grid1Selection');
+
+    describe('210 tutorial', function() {
+      xit('should output aria text for cells that come from selection feature', function () {
```

Comment:
You probably need a done here.
`it('should output aria text for cells that come from selection feature', function (done) {
 +          var focuser = element(by.css('.ui-grid-focuser'));
 +          grid1Selection.selectRow(1).then(function () {
 +            return gridTestUtils.click(focuser);
 +          })
 +          .then(function () {
 +             expect(element(by.css('.ui-grid-a11y-ariascreenreader-speakable.ui-grid-offscreen')).getInnerHtml()).toEqual('Row selected ');
 +             return focuser.sendKeys(protractor.Key.ARROW_DOWN);
 +          })
 +          .then(function () {
 +             expect(element(by.css('.ui-grid-a11y-ariascreenreader-speakable.ui-grid-offscreen')).getInnerHtml()).toEqual('Row not selected ');
 +             done();
 +          });
 +      });`